Doofinder/Feed/ApiClient/Client.php

```diff
@@ -121,7 +121,7 @@ public function get(string $path): string
      * @throws TypeAlreadyExists
      * @throws WrongResponse
      */
-    public function post(string $path, array $body): string
+    public function post(string $path, mixed $body): string
```

Comment:
Try not to use mixed in the params typing, since it is only allowed from PHP 8.0 (I was also about to use it, but I noticed this: https://www.php.net/manual/en/language.types.mixed.php)

```suggestion
    public function post(string $path, $body): string
```